### PR TITLE
OFFAPPS-779 extract brand strings

### PR DESCRIPTION
**Pull request by sandlerr**
https://zendesk.atlassian.net/browse/OFFAPPS-779

cc @zendesk/vegemite @zendesk/i18n

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -216,3 +216,11 @@ parts:
       key: "txt.apps.search_app.global.error.unavailable"
       title: "Error message when the search server is unavailable"
       value: "Sorry, we could not complete your search query. Please try again."
+  - translation:
+      key: "txt.apps.search_app.brand_filter.brand"
+      title: "title for dropdown: https://zendesk.box.com/s/hk3w166kbriae4jcj0zulc7l09kt98qs"
+      value: "Brand:"
```
Comment (mariasica) [inline translations/en.yml]: I made the link public. Also, we now have a screenshot field so url can be added there...not very very very terribly important, but we did it after feedback from Australia and now everyone implemented it... ;-)

**Comment by mariasica** [comment]
👍 for strings

File: translations/en.yml (hunk 1 above)
Comment (sandlerr) [inline translations/en.yml]: very cool, I was not aware. I'll update this.

**Comment by adammw** [comment]
:+1:

**Pull request merged**